test/api_spec.coffee

```diff
@@ -484,7 +486,11 @@ describe "api", ->
         true
       validate_presets = ->
         cloudinary.v2.api.upload_presets (error, response) ->
-          expect(response.presets.slice(0,3).map((p) -> p.name)).to.eql(delete_names)
+          count = delete_names.length
+          for preset in response.presets
+            if _.includes(delete_names, preset.name) && count > 0
+              count = count - 1;
+          expect(count).to.eql(0)
```

Comment:
Why?

Reply:
Otherwise, tests do not pass. Not always, but it did happen. The reason was that this code didn't delete everything after test's run.